content/en/blog/2022-05-04-april-update.md

```diff
@@ -0,0 +1,611 @@
+- Fri 20: [Observing Fastly's Network at Scale Thanks to K8s and the
+  Strimzi Operator - Fernando Crespo & Daniel Caballero,
+  (Fastly)](https://sched.co/ytrM) (11:00 - 11:35)
+- Fri 20: [Simplifying Service Mesh Operations with Flux and
+  Flagger - Mitch Connors (Google) &
+  Stefan Prodan (Weaveworks)](https://kccnceu2022.sched.com/#)
+  (14:55 - 15:30)
+
+Please note: all of the above might be subject to change. Please
+double-check the schedule beforehand. Please reach out to Vanessa
+Abankwah or Daniel Holbach on Slack if you have questions or would like
+to participate in any of the above.
+
+We very much look forward to seeing you there!
+
+### GitOps Days 2022
+
+GitOps Days 2022 is a free 2-day online event on June 8-9, 2022.
+
+This is **THE** event for your GitOps journey! Getting started? Taking
+GitOps to the next level? We'll cover all of the steps for your success!
+
+The event will run from **9:00 am PT to ~3:00 pm PT** each day as a free
+online event.
+
+✨✨ [Register
+now](https://go.weave.works/2022-GitOps-Days-US.html) to
+reserve your spot to receive updates to the schedule and speakers. ✨✨
+
+*Join the conversation!* Chat with the speakers and other attendees!
+Invite yourself at
+[https://slack.weave.works](https://slack.weave.works/)
+and hang out with us at
+[\#gitopsdays](https://bit.ly/GitOpsDays_Slack)
+
+What to expect?
+
+- Talks and tutorials on how to get started with Kubernetes and GitOps
+- Talks from Flux users about their use cases
+- How to do GitOps securely
+- Platforms that offer GitOps: Microsoft Arc Kubernetes, AWS Anywhere,
+  Weave GitOps, D2iQ Kubernetes Platform, and more! all using Flux!
+- Flux in the CNCF and the GitOps Ecosystem
+- Flux support and Integrations: Flux + Helm, Terraform, HashiCorp Vault,
+  Jenkins, OpenShift, Visual Studio Code, and much much more!
+- Technical deep dives with Flux maintainers
+- Music from DJ Desired State 🎶
+
+## In other news
+
+### People writing/talking about Flux
+
+We love it when you all write about Flux and share your experience,
+write how-tos on integrating Flux with other pieces of software or other
+things. Give us a shout-out and we will link it from this section! ✍
+
+#### Manage Kubernetes Secrets for Flux with HashiCorp Vault
+
+Rosemary Wang from HashiCorp wrote a great blog post about how to
+[manage Kubernetes Secrets for Flux with HashiCorp Vault](https://www.hashicorp.com/blog/manage-kubernetes-secrets-for-flux-with-hashicorp-vault). The how-to
+is nicely written with a lot of detail and will take you through the steps
+to configure the Secrets Store CSI driver with HashiCorp Vault to securely
+inject secrets into Flux or other GitOps tools on Kubernetes.
+
+We are looking forward to more collaboration together!
+
+#### Full GitOps Tutorial: Getting started with Flux CD
+
+{{< youtube 5u45lXmhgxA >}}
+
+This video is great for everyone who gets started, but also everyone who enjoys a story well-told.
+
+In this video, Anais Urlichs covers
```

Comment:
```suggestion
In this video, Anaïs Urlichs covers
```

Reply:
Yes, I wanted to do that, but that's not how it's spelled on the YouTube or website...

Reply:
![Bildschirmfoto von 2022-05-03 16-01-38](https://user-images.githubusercontent.com/1346979/166467794-42f2e89d-41e5-402b-848e-e9188a5e4529.png)